e2e/testcases/reconciler_manager_test.go

```diff
@@ -515,6 +528,25 @@ func TestAutopilotReconcilerAdjustment(t *testing.T) {
 		return nil
 	})
 
+	// manually increase first container CPU and memory request to above current user override request
```

Comment:
I feel this test can be combined with `e2e/testcases/override_resource_limits_test.go`: https://github.com/GoogleContainerTools/kpt-config-sync/pull/289.

This can be done in later PR.

Reply:
Do you suggest that we should combine the TestAutopilotReconcilerAdjustment test with TestOverrideReconcilerResourcesV1Alpha1? Currently we have TestOverrideReconcilerResourcesV1Alpha1 skip on Autopilot cluster. It looks like that override test is focusing on make sure the override function works and this  AutopilotReconcilerAdjustmentTest focusing on verify that adjustContainerResources works fine with resource override function. Combine these two tests will make a long complex test, we also need to manually check what's the resources result after autopilot adjustment for all the test cases in TestOverrideReconcilerResourcesV1Alpha1 to make a right prediction. So I think it is better to separate  these two tests.

Reply:
ACK